<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearningPopUp</class>
 <widget class="QDialog" name="LearningPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="learning_progress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>380</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="show_report">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show graph report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_weights">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save weights into file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold;</string>
   </property>
   <property name="text">
    <string>LEARNING PROGRESS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last epoch or group error:</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_error">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
